### DISCUSSION TITLE
[HELP] Toolbars Array for a Templater Script

### DISCUSSION THREAD

**Discussion by FelipeRearden** [chrisgurney/obsidian-note-toolbar#96] [Q&A] [answered]
Hello @chrisgurney !!!

I humble ask for a help.

I am building a templater script to update/add the notetoolbar yaml from a modal/suggester. 

This way, would you mind sharing with me the code where the output is an array with all the Note Toolbar name.
- One of the struggles of full mobile setup is that I cant access the console to get his info by hand; 🥲

📸 In other words: I need an array with this info;
![IMG_1256](https://github.com/chrisgurney/obsidian-note-toolbar/assets/85180696/a3922b9e-ca17-4384-8fad-371821f1ec91)

 <hr>

This is my code (remember that I am a newbie…)

```js

<%*
let array = ["standard","Books","Articles","Templater"];
const cmd = await tp.system.suggester(array, array)
const file = tp.file.find_tfile(tp.file.path(true));
//Frontmatter Update
await app.fileManager.processFrontMatter(file, (frontmatter) => {
  frontmatter["ntb"] = cmd;
})
%>
```

<hr>

Thank you very much 🙏 🙏 🙏

I wish you a fantastic day ☀️

**Comment by chrisgurney** [comment, marked as answer]
Hey @FelipeRearden have you looked at @FeralFlora's [script](https://github.com/chrisgurney/obsidian-note-toolbar/discussions/79)? 

Not only does the script get a list of toolbars from the plugin's settings, it sounds like it does what you're looking to build.

If it doesn't include what you're looking for, let me know!

**Reply by FelipeRearden** [reply]
Thank you very much Chris !!!

The script was perfect to solve my templater question !!!!

Many thanks to @FeralFlora for sharing the script 🙏

Thanks again and I wish you a fantastic Monday ☀️

**Reply by chrisgurney** [reply]
Indeed, thanks again for sharing your script @FeralFlora !
